# Generated for NaoQi method ALMemory.getExtractorEvent
# Get event list generated by extractor

# Name of extractor that create event
string name
---
# return an array of data's string name.
string[] value


